web/static/js/meetup.js

```diff
@@ -34,6 +34,9 @@ $(function() {
     dataType: "jsonp",
   }).done(function(data) {
     $.each(data.data, function(index, meetup) {
+      $("#meetup-placeholder").fadeOut("medium", function() {
+        $("#meetups").fadeIn("medium");
+      })
```

Comment:
Generally I enjoy marrying javascript DOM lookups by a `data` attributes instead of an `id` or `class`, this way if another designer comes in and changes the styling, it won't affect functionality of the javascript.

Reply:
Shouldn't the designer never touch the ID? ID's are so much faster than `data-attr` that I think the tradeoff is worth it https://jsperf.com/jquery-selectors-class-vs-data-attribute-vs-id

Reply:
hard to argue with that. 👍